crates/meroctl/src/cli/context/create.rs

```diff
@@ -110,44 +120,48 @@ async fn create_context(
 
 async fn watch_app_and_update_context(
     base_multiaddr: &Multiaddr,
-    context_id: calimero_primitives::context::ContextId,
+    context_id: ContextId,
     path: Utf8PathBuf,
     client: &Client,
     metadata: Option<Vec<u8>>,
-) -> eyre::Result<()> {
+) -> EyreResult<()> {
     let (tx, mut rx) = mpsc::channel(1);
 
-    let handle = tokio::runtime::Handle::current();
+    let handle = Handle::current();
     let mut watcher = notify::recommended_watcher(move |evt| {
         handle.block_on(async {
-            let _ = tx.send(evt).await;
-        })
+            drop(tx.send(evt).await);
```

Comment:
Good practice. For a type with a non-trivial `Drop` implementation, this causes it to be dropped right away instead of waiting until the end of scope. In longer functions this can have more of an effect, but enabling the lint means we do it everywhere. There's no downside - only upside.

https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#let-underscore-drop